test/functional/readme.md

```diff
@@ -62,40 +62,44 @@ cd test/functional
 ```
 
 This last command contains a script for running tests on all browsers.
```

Comment:
Is this still possible? Otherwise we should mention that the tests have to run for each browser individually by specifying the target browser

Reply:
Yes, I need to review this part